components/chef-ui-library/src/molecules/chef-phat-radio/chef-phat-radio.tsx

```diff
@@ -47,6 +47,14 @@ export class ChefPhatRadio {
       this.value = this.selected.value;
     }
     this.selected.selected = true;
+
+    // add keypress listeners to all the child options
+    const theseOptions = this.el.querySelectorAll(`chef-option`);
+    theseOptions
+      .forEach(option => {
+        option.addEventListener('keypress', event => { this.handleKeypress(event); });
```

Comment:
The way you suggested is how I would generally do it.  But what i've found is that in this setup, if I don't specifically pass the event, The element function does not receive it.  I was able clean this up a little bit by removing the curly braces. 

My hunch is that maybe its because this element is nested an extra level down, but I actually cant use `option.addEventListener('keypress', this.handleKeypress);`,  as it won't work.  so I've opted for `option.addEventListener('keypress', event => this.handleKeypress(event));`